io/GDALReader.hpp

```diff
@@ -58,6 +58,38 @@ class PDAL_DLL GDALReader : public Reader , public Streamable
     ~GDALReader();
 
 private:
+    class BlockReader
+    {
+    public:
+        BlockReader(GDALReader& reader);
+        void initialize();
+        point_count_t processBlock(PointViewPtr view);
+        bool processOne(PointRef& point);
+
+    private:
+        struct Block
+        {
+            std::vector<std::vector<double>> m_data;
+            int m_blockCol;
+            int m_blockRow;
+        };
+
+        bool readBlock();
+
+        GDALReader& m_reader;
+        int m_blockRow;
```

Comment:
Please add a TODO to hide these args like we do with some of the other drivers.

Reply:
I'm having a hard time finding a similar TODO. Want to keep the comment consistent. Not exactly sure what hide means in this case.

Reply:
Mostly a note to myself. I'll take care of it